packages/common/src/utils/formatting.ts

```diff
@@ -15,17 +13,17 @@ export const formatDurationTag = (seconds: number): string | null => {
  * Hours are only shown if at least 1
  * Minutes get rounded
  *
- * @returns string, such as '2h 24m' or '31m'
+ * @returns string, such as '2hrs 24min' or '31min'
  */
 
-export const formatDuration = (duration: number): string | null => {
-  if (!duration) return null;
+export const formatDuration = (duration: number) => {
+  if (!duration) return '';
 
   const hours = Math.floor(duration / 3600);
   const minutes = Math.round((duration - hours * 3600) / 60);
 
-  const hoursString = hours ? `${hours}h ` : '';
-  const minutesString = minutes ? `${minutes}m ` : '';
+  const hoursString = hours ? `${hours}hrs ` : '';
```

Comment:
Should these be translations?

Reply:
Yes, but we found more similar cases that need to be updated. This quick fix is an improvement because instead of screenreaders reading 10 meters, it now reads 10 minutes in English and Spanish.

See this discussion: https://github.com/Videodock/ott-web-app/pull/116#discussion_r1490592227

We have made a ticket for this, but it's not a high-priority ticket.